tests/xml/test_files/xsd-error.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0"?>
```

Comment:
If I understand correctly, the stdout (which gives corrected output) adds it to the corrected one.
Hence, for coverage of Line 36.

But I havent followed this very closely. So, would be good to let @vivek425ster confirm